H/tests/tests.hs

```diff
@@ -73,5 +73,6 @@ main :: IO ()
 main = do
     -- Normalize internalization settings.
     setEnv "LANG" "C"
+    setEnv "LC_ALL" "C"
```

Comment:
Good to know.

The problem with UTF-8 here is that C.UTF-8 is a Debian thing. Other distros, such as Arch Linux, don't have it. We have tests with unicode elsewhere.